README.md

```diff
@@ -2,7 +2,7 @@
 
 A Docker logging driver plugin to send logs to Sumo Logic.
 
-**Disclaimer:** This plugin is still being developed.  We recommend using this plugin in non-production environments.
+**Disclaimer:** This plugin is still being developed. We recommend using this plugin in non-production environments.
```

Comment:
If we are announcing this at Illuminate we should remove this warning. (can be separate PR)